pylint/checkers/strings.py

```
@@ -925,6 +933,69 @@ def _detect_u_string_prefix(self, node: nodes.Const):
+                    i[0] for i in utils.parse_format_method_string(node.value)[0]
+                ]
+                for keyword in node.parent.parent.keywords:
+                    # If keyword is used multiple times
+                    if keyword_args.count(keyword.arg) > 1:
+                        return
+
+                    keyword = utils.safe_infer(keyword.value)
+
+                    # If lists of more than one element are being unpacked
+                    if isinstance(keyword, nodes.Dict):
+                        if len(keyword.items) > 1 and len(keyword_args) > 1:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)